x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/alerts_cell_actions.cy.ts

```diff
@@ -74,6 +74,7 @@ describe('Alerts cell actions', { tags: ['@ess', '@serverless'] }, () => {
 
     cy.log('filter out alert property');
 
+    scrollAlertTableColumnIntoView(ALERT_TABLE_FILE_NAME_HEADER);
```

Comment:
Thank for taking a look @logeekal! 😊

Technically nothing should have changed since this PR only affects the way we fetch alerts and not the surrounding API or UI surface although there was a bit of state management in the previous implementation which might have had a slightly different behavior to what is now managed by react-query directly.
I don't have a clear idea on why it happens only now, but when running the test locally I could see it timing out because the relevant column was far to the right of the table and therefore it was not rendered by the table's virtual scroller.

Reply:
Thanks for explanation @umbopepato , Is it also possible the order of the columns has changed which led `file.name` to go more to the right? 

Let me quickly check it.